### PR TITLE
Added filter to limit TTS to voice channel members.

### PR DESCRIPTION
**Pull request by sithsiri**
### Changes

- [x] New Feature(s)
- [ ] Bug Fixes
- [ ] Documentation
- [ ] Other: \_____

## Description

Makes sure a member is actually in the voice channel before proceeding with TTS.

### CODE REVIEW

**Comment by sithsiri** [comment]
Please see the commit first, as I have a feeling my ultra-janky solution isn't an ideal solution.

File: index.js [hunk 1]
```diff
@@ -37,12 +37,20 @@ function record(voiceConnection) {
...
+    return false;
+}
+
 function tts(message) {
     if (getMe(message.channel.guild).voiceState == null) {
         playDict.splice(playDict.indexOf(message.channel));
         return;
     }
     voiceConnection = client.voiceConnections.get(message.channel.guild.id);
+    if (!collectionValuesContain(voiceConnection.channel.members, message.author.fetchMember()) return;
```
Comment (Sei4or) [inline index.js]:
```suggestion
    if (!message.channel.guild.channels.get(getMe(message.channel.guild).voiceState.channelID).members.has(message.author.id)) return;
```
This is a _better_ solution to only including connected members.

File: index.js [hunk 2]
```diff
@@ -37,12 +37,20 @@ function record(voiceConnection) {
     voiceDataStream.on("error", console.error);
 }
 
+function collectionValuesContain(c, o) {
```
Comment (Sei4or) [inline index.js]: This function is obsolete due to my other change.

**Review by Sei4or — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (sithsiri) [reply index.js]: *such better*

File: index.js (hunk 2 above)
Comment (sithsiri) [reply index.js]: I will redact it as it seems utterly useless.

File: index.js (hunk 1 above)
Comment (Sei4or) [reply index.js]: Well, it is better in the sense that it will work.

**Review by Sei4or — APPROVED** [review]
yes

**Pull request merged**